project-management-openproject/project-management-openproject-ui/src/main/resources/ProjectManagement/ProjectManagementAdministration.xml

```diff
@@ -37,8 +37,8 @@
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
   <content>{{velocity}}
-==OpenProject Instance Connections==
-Defines access to OpenProject instances and associates a Connection name to each. The Connection name can then be used in various Open Project components whenever an OpenProject instance is required, such as in a parameter for the OpenProject Macro.
+==$escapetool.xml($services.localization.render('openproject.config.connectionsTitle'))==
```

Comment:
In this scenario, you want to escape the translation for xwiki syntax.

You are inside a xwiki/2.1 context and we want to make sure no syntax is being injected.

Example here https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-attachment/xwiki-platform-attachment-ui/src/main/resources/XWiki/AttachmentSelector.xml#L148-L149